operantanalysis/operantanalysis.py

```diff
@@ -156,3 +157,37 @@ def lever_press_latency(timecode, eventcode, lever_on, lever_press):
         return round(statistics.mean(press_latency), 3)
     else:
         return "No presses"
+
+
+def total_head_pokes(eventcode):
+    """
+
+    :param eventcode: list of event codes from operant conditioning file
+    :return: total number of times animal poked head into reward receptacle
+    """
+    return eventcode.count("PokeOn1")
+
+
+def num_successful_go_nogo_trials(eventcode):
+    """
+
+    :param eventcode: list of event codes from operant conditioning file
+    :return: number of successful go and no go trials in the go/no go tasks
+    """
+    return eventcode.count('SuccessfulGoTrial'), eventcode.count('SuccessfulNoGoTrial')
+
+
+def count_go_nogo_trials(eventcode):
+    """
+
+    :param eventcode: list of event codes from operant conditioning file
+    :return: number of go and no go trials in the go/no go tasks
+    """
+    lever_on = [i for i, event in enumerate(eventcode) if event == 'RLeverOn' or event == 'LLeverOn']
+    (go_trials, nogo_trials) = (0, 0)
+    for lever in lever_on:
+        if eventcode[lever + 1] == 'LightOn1' or eventcode[lever + 1] == 'LightOn2':
```

Comment:
```suggestion
        if eventcode[lever + 1] in ('LightOn1', 'LightOn2'):
```
?  otherwise I was staring at the screen first trying to see if there is difference in the offset (`lever + 1`).  This way it makes it more explicit.  Or even you don't care which `LightOn` then even
```suggestion
        if eventcode[lever + 1].startswith('LightOn'):
```
?

Reply:
I added this change in new branch!